libraries/cyclestreets-view/src/main/java/org/mapsforge/android/maps/MapsforgeOSMDroidTileProvider.java

```diff
@@ -19,6 +19,8 @@
 import okhttp3.Request;
 import okhttp3.Response;
 
+import static org.osmdroid.tileprovider.constants.OpenStreetMapTileProviderConstants.*;
```

Comment:
The only reason for this import static is that previously fields such as `TILE_DOWNLOAD_MAXIMUM_QUEUE_SIZE` were a part of the superclass.  In the latest version of `osmdroid` they have been abstracted out to a separate constants class, and the static wildcard import seemed the cleanest way to continue incorporating them.

I don't see any way that this static import could affect the tileserver URL that we're using.